apps/api/src/app/order/order.service.ts

```diff
@@ -123,10 +123,29 @@ export class OrderService {
       data.SymbolProfile.connectOrCreate.create.assetSubClass = assetSubClass;
       data.SymbolProfile.connectOrCreate.create.dataSource = dataSource;
       data.SymbolProfile.connectOrCreate.create.name = name;
-      data.SymbolProfile.connectOrCreate.create.symbol = id;
+
+      let symbolProfileSymbol = id;
+
+      if (data.type === 'INTEREST') {
+        const symbolProfile = await this.prismaService.symbolProfile.findFirst({
+          orderBy: {
```

Comment:
Probably not necessary for `findFirst`

Reply:
If `where` is unique (see my comment on collision below), I think `orderBy` is not needed